### DISCUSSION TITLE
CR-10 SE profile

### DISCUSSION THREAD

**Discussion by DragonGyld** [OrcaSlicer/OrcaSlicer#3609] [Ideas]
Will the Cr-10 SE be added into orca at some point?

**Comment by Adilrafiq001** [comment]
Use this till Orca gives us the official profile

https://www.printables.com/model/709820-creality-cr10-se-orca-slicer-profile

**Reply by DenisLutsky** [reply]
This one requires tuning I've just printed OrcaCude and here's the result. I just changed the filament to the "Default" one. 
And there is no issues in slicer, it's not showing these lines. 
![photo_2024-02-15_12-17-50](https://github.com/SoftFever/OrcaSlicer/assets/116076603/3b5d8842-e568-4f31-888b-743201f0431e)
![photo_2024-02-15_12-17-46](https://github.com/SoftFever/OrcaSlicer/assets/116076603/d941b9f7-ebab-4c06-82a9-2ad83e0bc7a6)
I'm new to 3D printing and maybe just don't understand something

**Reply by DenisLutsky** [reply]
It prints fast, and clean, but thread and screw failed. 
And the problem with that config is that it's heating plate first and only then starts heating extruder. And also it's not parking plate and head after print finished, you can see that burn on the left bottom of the Z face. That's because it's stuck after printing was finished
